lib/elixir/lib/option_parser.ex

```diff
@@ -114,6 +117,31 @@ defmodule OptionParser do
   end
 
   @doc """
+  The same as `parse/2` but raises an `ArgumentError` if it fails.
+  Returns the parsed values, remaining arguments and an empty list with the
+  invalid options otherwise.
```

Comment:
Nevermind about `parse/2`, it has already been updated in https://github.com/elixir-lang/elixir/pull/4034.